webpack/assets/javascripts/react_app/components/FactCharts/FactChart.fixtures.js

```diff
@@ -0,0 +1,36 @@
+import Immutable from 'seamless-immutable';
+import { STATUS } from '../../constants';
+import { mockStoryData } from '../common/charts/DonutChart/DonutChart.fixtures';
+
+export const chartDataValues = mockStoryData.data.columns;
+
+const state = {
+  modalToDisplay: {},
+  chartData: [],
+  loaderStatus: '',
+};
+
+export const initialState = Immutable(state);
+
+export const modalOpenState = initialState.setIn(['modalToDisplay'], {
+  1: true,
+});
+
+export const modalSuccessState = initialState
+  .setIn(['modalToDisplay'], {
+    1: true,
+  })
+  .setIn(['chartData'], chartDataValues)
```

Comment:
you can use `state.merge` when it comes to multiple keys.

Reply:
Thanks!